.github/workflows/nuget-preview.yml

```diff
@@ -0,0 +1,66 @@
+name: GitHub Actions
+
+on:
+  schedule:
+  - cron: "5 4 3 */1 *" # once a month, at 4:05 on 3rd
+
+jobs:
+  release:
+    name: Release Preview
+    runs-on: ubuntu-latest
+    environment: NuGet
+    timeout-minutes: 10
+
+    env:
+      PYTHONNET_SHUTDOWN_MODE: Normal
+
+    steps:
+      - name: Get Date
+        run: |
+          echo "DATE_VER=$(date "+%y-%m-%d")" >> $GITHUB_ENV
```

Comment:
`%Y-%m-%d`

Reply:
I think two year digits is fine, unless you believe the project will live 80 years with no changes to preview release process.

Reply:
It's more to get a ISO8601 date, just because I like standards ;)

Reply:
Fine. I will also be a bit more apparent then.